app/page/trip-map.cjsx

```diff
@@ -0,0 +1,41 @@
+React                  = require 'react'
```

Comment:
This is a general question: Should we make this kind of pages as own pages or like on Route page, where there is only one page?